docs/pages/build-reference/variants.mdx

```diff
@@ -79,13 +79,31 @@ export default {
   },
   android: {
     package: IS_DEV ? 'com.myapp.dev' : 'com.myapp',
-  },
+  }
 };
 ```
 
 In the above example, the environment variable `IS_DEV` is used to differentiate between the development and production environment. Based on its value, the different Application IDs or Bundle Identifiers are set for each variant.
 
-> **Note**: If you are using any libraries that require you to register your application identifier with an external service to use the SDK, such as Google Maps or Firebase Cloud Messaging (FCM), you'll need to have a separate configuration for that API for the `android.package` and `ios.bundleIdentifier`. You can also swap this configuration using the same approach as above.
+#### Additional app variant customizations
+
+You may wish to customize other aspects of your app on a per-variant basis. You can swap any configuration that you used previously in **app.json** using the same approach as above.
+
+Some examples:
+
+- If you are using any libraries that require you to register your application identifier with an external service to use the SDK, such as Google Maps or Firebase Cloud Messaging (FCM), you'll need to have a separate configuration for that API for the `android.package` and `ios.bundleIdentifier`.
+- If you are using [development builds](/develop/development-builds/introduction/) you can set the `expo-dev-client` config plugin to disable the app scheme used by Expo CLI and EAS Update QR codes on your non-development build, so those URL's will always launch the development build regardless of your device defaults:
```

Comment:
```suggestion
- - If you're using [development builds](/develop/development-builds/introduction/), you can configure the `expo-dev-client` plugin to disable the app scheme used by Expo CLI and EAS Update QR codes in non-development builds. This ensures that those URLs will always launch the development build, regardless of your device's defaults:
```

This is a very long sentence and hard to read. Suggestion to improve the readability
- Split it into two sentences
- "device defaults" sounds something is missing or don't provide enough context to what those defaults are being referenced to? Is it the default settings of the device? I don't have much context so I'm just making an assumption here.
- In the first sentence, it will be good to mention which property I can use to disable the app scheme and mention that property name. Often time, if we provide the context in written form of a property or a config along with a code example, it adds clarity for the reader to understand the context.
- Using one pattern of contraction (`you're` or `you are`) since this is the same paragraph